skimage/filters/tests/test_gaussian.py

```diff
@@ -62,8 +64,24 @@ def test_multichannel():
 
 
 def test_preserve_range():
+    """Test preserve_range parameter."""
+    ones = np.ones((2, 2), dtype=np.int64)
+    filtered_ones = gaussian(ones, preserve_range=False)
+    assert np.all(filtered_ones == filtered_ones[0, 0])
+    assert filtered_ones[0, 0] < 1e-10
+
     img = np.array([[10.0, -10.0], [-4, 3]], dtype=np.float32)
-    gaussian(img, 1, preserve_range=True)
+    gaussian(img, 1)
+
+
+def test_1d_ok():
+    """Testing Gaussian Filter for 1D array.
+    With any array consisting of positive integers and only one zero - it should filter all values
+    to be at least more than 0.1
```

Comment:
"at least" and "more than" are redundant :wink: 
I recommend going for the mathematical standard way of saying (i.e., "greater than")